contracts/test/AllowanceTarget/CompleteSetSpender.t.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.7.6;
+
+import "contracts-test/AllowanceTarget/Setup.t.sol";
+
+contract TestAllowanceTargetCompleteSetSpender is TestAllowanceTarget {
+    function testCannotSetBeforeSetNew() public {
+        vm.expectRevert("AllowanceTarget: no pending SetSpender");
+        allowanceTarget.completeSetSpender();
+    }
+
+    function testCannotSetTooEarly() public {
+        allowanceTarget.setSpenderWithTimelock(newSpender);
+        vm.expectRevert("AllowanceTarget: time lock not expired yet");
+        allowanceTarget.completeSetSpender();
+    }
+
+    // normal case
```

Comment:
I think this comment doesn't transmit too much info. Maybe we could just remove it

Reply:
OK! Is it only in this CompleteSetSpender.t.sol file?
Or should I remove all "normal case" comments in all *.t.sol contract files?